src/finfield.c

```diff
@@ -1020,7 +1020,7 @@ Obj             SumFFEInt (
     FFV                 vL, vR, vX;     /* value of left, right, result    */
     FF                  fX;             /* field of result                 */
     Int                 pX;             /* char. of result                 */
-    FF*                 sX;             /* successor table of result field */
```

Comment:
What's FF vs FFV? Is it related to constness?

Reply:
`FF` is the "type of small finite fields", while `FFV` is "type of the values of elements of small finite fields". So using the former here was always wrong; however, since both are typedef'ed to `UInt2`, this didn't (and doesn't) matter. But I figured that if I touch that line anyway, I might as well change it to use the correct type :-).